modules/persistent-storage-csi-vsphere-change-max-snapshot.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-vsphere.adoc
+// * storage/persistent_storage/persistent-storage-csi-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="vsphere-change-max-snapshot_{context}"]
+= Changing the maximum number of snapshots for vSphere
+
+The default maximum number of snapshots per volume in vSphere Container Storage Interface (CSI) is 3. You can change the maximum number up to 32 per volume. 
+
+However, be aware that increasing the snapshot maximum involves a performance trade off, so for better performance use only 2 to 3 snapshots.
+
+For more VMWare snapshot performance recommendations, see *_Additional Resources_*.
+
+.Prerequisites
+
+* Access to the cluster with administrator rights.
+
+.Procedure
+
+. Check the current config map by the running the following command:
```

Comment:
We'd better also mention that current example shows no `maximum number of snapshots` configured, it will use the default number `3`.

Reply:
Done.